src/components/common.js

```diff
@@ -505,30 +505,28 @@ export const HeroWrapper = ({ showMenu = true, bigSubhead = false, showDocLink =
         backgroundRepeat: 'no-repeat', backgroundSize: '750px', backgroundPosition: 'right 0 top 0'
       }
     }, [
-      div({ style: { maxWidth: '50%' } }, [
-        h1({ style: { fontSize: 54 } }, [`Welcome to ${getAppName({ longName: isTerra() })}`]),
-        div({ style: { margin: '1rem 0', width: 700, ...(bigSubhead ? { fontSize: 20, lineHeight: '28px' } : { fontSize: 16, lineHeight: 1.5 }) } }, [
-          `${getAppName({ longName: !isTerra(), capitalizeThe: true })} is a ${Utils.cond(
-            [isTerra(), () => 'cloud-native platform'],
-            [isFirecloud(), () => 'NCI Cloud Resource project powered by Terra'],
-            [isProjectSingular(), () => 'project funded by Additional Ventures and powered by Terra'],
-            [isRareX(),
-              () => 'federated data repository of rare disease patient health data, including patient reported outcomes, clinical and molecular information. The platform is powered by Terra'],
-            () => 'project powered by Terra'
-          )} for biomedical researchers to `,
-          heavyWrapper('access data'), ', ', heavyWrapper('run analysis tools'), ', ',
-          span({ style: { whiteSpace: 'nowrap' } }, ['and', heavyWrapper(' collaborate')]),
-          isProjectSingular() && ' to advance research around single ventricle heart disease',
-          '. ',
-          showDocLink ?
-            h(Link, {
-              style: { textDecoration: 'underline' },
-              href: `https://support.terra.bio/hc/en-us`,
-              ...Utils.newTabLinkProps
-            }, ['Learn more about Terra.']) : null
-        ]),
-        children
-      ])
+      h1({ style: { fontSize: 54, width: 'calc(100% - 460px)' } }, [`Welcome to ${getAppName({ longName: isTerra() })}`]),
+      div({ style: { margin: '1rem 0', width: 'calc(100% - 460px)', maxWidth: 700, ...(bigSubhead ? { fontSize: 20, lineHeight: '28px' } : { fontSize: 16, lineHeight: 1.5 }) } }, [
```

Comment:
Would it be good to have one source for the value `460` and add a comment about why that value was chosen? I feel it may be too easy to modify it in one place and not the other, and not notice the consequences.

Reply:
Added a comment in https://github.com/DataBiosphere/terra-ui/pull/3227/commits/4b808727f5e72865d58677f47b57feaf0b2794e0.